ArticleService.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+const URL = 'https://panda-market-api-crud.vercel.app/articles/';
+
+export async function getArticleList({page, pageSize, keyword}) {
+  return fetch(URL,{
+      params: {page, pageSize, keyword}
+    })
+  .then(res => res.json())
+  .then(data => {
+    return data
+  })
+  .catch((e) => {
+    if(!res.ok) {
+    console.log('목록을 가져오는 중 오류가 발생했습니다.');
+  }
+});
+}
+
+export async function getArticle(articleId) {
+  const res = await axios.get(`${URL}${articleId}`)
+  .catch((e) => {
+    console.log('오류가 발생했습니다.');
+    return null;
+  });
+  return res?.data || null;
+}
+
+export async function createArticle({title, content, image}) {
+  const res = await axios.post(
+  `${URL}`, 
+  {title, content, image}
+  )
+  .catch((e) => {
+    console.log('오류가 발생했습니다.');
+    return null;
+  });
+  return res?.data || null;
+}

```

Comment:
await은 비동기 함수를 동기적으로 동작하게 해주지만, 개발자가 문맥적으로도 동기적으로 `기다린다` 라는 시각으로 바라보기도 합니다.
비동기 방식의 체이닝 방식으로 .catch로 에러를 제어해주셨는데, 일반적으로 async/await에서 에러 처리는 try/catch로 작성하시는 게 자연스럽습니다.

```suggestion
export async function createArticle({title, content, image}) {
  try {
    const res = await axios.post(
      `${URL}`, 
      {title, content, image}
    )
    return res.data;
  } catch (e) {
    console.log('오류가 발생했습니다.');
    return null;
  }
}

```